src/main/java/com/edu/ComNumber.java

```diff
@@ -0,0 +1,27 @@
+package com.edu;
+
+public class ComNumber {
+
+    public static final int COM_NUMBER_SIZE = 3;
+    public static final int MAX_COM_NUMBER_RANDOM = 9;
+    public static final int COM_NUMBER_START = 1;
+
+    public String comNumCreate() {
+
+        String comNumStr = "";
+        int comNum = 0;
+
+        for (int i = 0; i < COM_NUMBER_SIZE; i++) {
+            comNum = (int) (Math.random() * MAX_COM_NUMBER_RANDOM) +
+                    COM_NUMBER_START;
```

Comment:
```suggestion
            comNum = (int) (Math.random() * MAX_COM_NUMBER_RANDOM) + COM_NUMBER_START; 
```
굳이 COM_NUMBER_START 를 띄울 필요는 없다고 생각됩니다. (화면에 충분히 들어오는 것 같아 불필요하게 느껴집니다) 

또한 conNum을 **for문 밖에서 변수선언**을 하고 값을 재할당해서 사용하고 있는데, ``comNum`` 변수는 for문 내부에서만 사용되는 것으로 보입니다. 변수선언을 for문 내부에서 값을 할당함과 동시에 사용하는 것이 더 좋을 것 같습니다.